### PR TITLE
Update RewardableValidators to be compatible with BridgeValidators  

### LINKED ISSUE #168: RewardableValidators event definitions causes additional complexity
See this thread https://github.com/poanetwork/bridge-ui/pull/184#discussion_r274034802

### PR DESCRIPTION
**Pull request by patitonar**
Closes #168

### CODE REVIEW

File: contracts/upgradeable_contracts/RewardableValidators.sol [hunk 1]
```diff
@@ -54,11 +54,11 @@ contract RewardableValidators is BaseBridgeValidators {
         return isInitialized();
     }
 
-    function addValidator(address _validator, address _reward) external onlyOwner {
+    function addRewardableValidator(address _validator, address _reward) external onlyOwner {
```
Comment (akolotov) [inline contracts/upgradeable_contracts/RewardableValidators.sol]: We need to redefine `addValidator` in this contract as so it cause `revert()` if it is called. Such redefinition will keep data consistency if it is called occasionally.

Comment (patitonar) [reply contracts/upgradeable_contracts/RewardableValidators.sol]: I'm not sure I understood this correctly. `addValidator(address _validator)` method is defined on `BridgeValidators` contract, not `BaseBridgeValidators`.  So the only method to add a validator on `RewardableValidators` contract is `addRewardableValidator(address _validator, address _reward)`

Comment (akolotov) [reply contracts/upgradeable_contracts/RewardableValidators.sol]: Yes, you are correct! sorry for misunderstanding.

File: contracts/upgradeable_contracts/RewardableValidators.sol [hunk 2]
```diff
@@ -5,7 +5,7 @@ import "./BaseBridgeValidators.sol";
 
 contract RewardableValidators is BaseBridgeValidators {
 
-    event ValidatorAdded (address indexed validator, address reward);
+    event ValidatorAdded (address indexed validator);
```
Comment (akolotov) [inline contracts/upgradeable_contracts/RewardableValidators.sol]: Do you think it make sense to move declarations of both events to the BaseBridgeValidators contract?

Comment (patitonar) [reply contracts/upgradeable_contracts/RewardableValidators.sol]: Good catch! Yes, I think it makes sense. Done https://github.com/poanetwork/poa-bridge-contracts/pull/169/commits/9be3f38bdf2fa584a0b1367154abf33a6757ad99

**Pull request merged**